systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/ListenersST.java

```diff
@@ -668,6 +668,100 @@ void testLoadBalancerTls(ExtensionContext extensionContext) {
         );
     }
 
+    private String clusterIpBootstrapServiceName(String namespaceName, String clusterName, String listenerName) {
+        String serviceName = String.format("%s-kafka-%s-bootstrap", clusterName, listenerName);
+        String bootstrapServiceName = String.format("%s.%s.svc", serviceName, namespaceName);
+        return bootstrapServiceName;
+    }
```

Comment:
Okay, this looks good.
Can you move it to `KafkaUtils` please?
Also, do you think that it will be same for every environment? As I mentioned in previous comment - to be sure that the name will be correct, we are getting it from the Kafka status (you can check it in `AbstractKafkaClient` in `getBootstrapServerFromStatus` method). Do you think it will be better to get if from status, to prevent some future issues with different bootstrap name?

Reply:
Yes! Sure!
I can add a similar method to KafkaClient, or make KafkaClient implementing AbstractKafkaClient (there is a reason why internalClients.KafkaClient didn't implement AbstractKafkaClient?) to take advantage of this class

Reply:
something like
 `    public String computeBootstrapAddressFromStatus(String clusterName, String listenerName) { 
//same code as AbstractKafkaClient.getBootstrapServerFromStatus }
` 
returning bootstrapAddress, which can be used to set bootstrapAddress field on client. The method can be added to internalClients.KafkaClient or internalClients.BaseClient to be used by other clients (bridge, ...)

what do you prefer?

Reply:
I think that it would be out of scope of this PR, I think in this PR you can just add that method to the `KafkaUtils` and use it in those two tests.
External and internal clients were a bit different in the past, I think we can make it more abstract in the future.
Anyway, thanks for the changes :)

Reply:
done.

ListenersST test run successfully in my local env

thanks for your review !